sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -16,3 +16,30 @@ export interface GetApplicationDataDto {
    */
   data: any;
 }
+
+export interface ApplicationAssessmentDTO {
```

Comment:
IMO, since all this data will be flexible, I would not mind not express all individual properties.
If all the data will be consumed only by a Form.IO form and only Form.IO form will know the expected fields, I would say to express it as flexible as possible.

Reply:
I feel it's better to fetch it individually, as its easy to show case in the VUE application. Anyhow, when we need to show additional columns, we need to do a code change irrespective on the form side.

Reply:
Not quite agree. When we need more column/data, we can change the workflow to save more data and we can change the Form.IO definition, no Vue code change is needed at all.
Making those expressed in this way seems to be against what we talked about in the meeting where we defined it.